src/pcapi/scripts/offerer/import_offerer_from_csv.py

```diff
@@ -0,0 +1,119 @@
+import ast
+import csv
+from typing import Dict
+
+from pcapi.core.offerers.api import create_digital_venue
+from pcapi.core.offerers.models import Venue
+from pcapi.core.users.api import create_pro_user
+from pcapi.models import ApiErrors
+from pcapi.models import Offerer
+from pcapi.models import VenueType
+from pcapi.repository import repository
+from pcapi.repository.offerer_queries import find_by_siren
+from pcapi.repository.user_queries import find_user_by_email
+from pcapi.routes.serialization.users import ProUserCreationBodyModel
+from pcapi.utils.logger import json_logger
+
+
+def create_offerer_from_csv(row: Dict) -> Offerer:
+    offerer = Offerer()
+    offerer.name = row["nom_structure"] if row["nom_structure"] else row["Name"]
+    offerer.siren = row["SIREN"]
+    offerer.address = row["adresse"].split(",")[0]
+    offerer.postalCode = _get_postal_code(row)
+    offerer.city = row["City"]
+
+    return offerer
+
+
+def create_venue_from_csv(row: Dict, offerer_siren: str) -> Venue:
+    venue = Venue(
+        address=row["adresse"].split(",")[0],
+        postalCode=_get_postal_code(row),
+        city=row["commune"],
+        departementCode=row["Département"],
+        siret=row["SIRET"].replace(".0", ""),
+        publicName=row["nom_lieu"],
+        bookingEmail=row["Email"],
+        venueType=VenueType.query.filter_by(label=row["Catégorie"]).first(),
+    )
+
+    if row["nom_lieu"]:
+        venue.name = row["nom_lieu"]
+    else:
+        venue.name = (
+            f"Lieu {Venue.query.filter(Venue.siret.startswith(offerer_siren)).count() + 1} - {row['nom_structure']}"
+        )
+
+    try:
+        geoloc = ast.literal_eval(row["geoloc"])
+        venue.latitude = geoloc[0]
+        venue.longitude = geoloc[1]
+    except SyntaxError:
+        pass
```

Comment:
J'ai l'impression que les données ressemblent fortement à du JSON. Même si le CSV est généré par nous, je pense qu'utiliser `json.loads()` serait plus raisonnable. :)

```suggestion
    if row["geoloc"]:
        geoloc = json.loads(row["geoloc"])
        venue.latitude, venue.longitude = geoloc
```